pkg/agent/openflow/pipeline.go

```diff
@@ -42,73 +42,73 @@ import (
 
 var (
 	// PipelineClassifierTable is table 0. Packets are forwarded to different pipelines in this table.
-	PipelineClassifierTable = newFeatureTable("PipelineClassifier", 0xff, 0)
+	PipelineClassifierTable = newTable("PipelineClassifier", 0, binding.PipelineAll)
 
 	// OVS pipeline for ARP is used to process ARP packets.
 	// Tables in ValidationStage:
-	ARPSpoofGuardTable = newFeatureTable("ARPSpoofGuard", 0x7f, binding.ValidationStage)
+	ARPSpoofGuardTable = newTable("ARPSpoofGuard", binding.ValidationStage, binding.PipelineARP)
```

Comment:
We should add a comment saying the order of declaring the tables decides the order of the tables in the stage.

Reply:
Where do we introduce the "stage" concept? Which file?

I think the stages should be declared in pkg/agent/openflow, but not in pkg/ovs/openflow

Reply:
Have you missed my questions / comments here?

Reply:
In my opinion, stage is like a part of pipeline. Pipeline like `binding.PipelineARP`, `binding.PipelineIP` is defined in `pkg/ovs/openflow`, and stage should be also defined there.

Reply:
I thought pkg/ovs/openflow are for common OVS/openflow code, not specific to Antrea pipeline implementation, which should go pkg/agent/openflow.

And could you answer this question: "Where do we introduce the "stage" concept? Which file?"? I just want to check how we describe "stage" in the comments.